templates/go/manifest.yaml

```diff
@@ -0,0 +1,5 @@
+# Optional list of additional Buildpacks to be applied to the language pack's
+# builder image when the Function is built using a Buildpack builder.
+buildpacks:
+  - paketo-buildpacks/go-dist
+  - ghcr.io/boson-project/go-function-buildpack:tip
```

Comment:
@lance just to be sure, is this OK? It seems redundant with `DefaultBuilderImages` map above.

Reply:
Yes - this is the same as in the upstream: https://github.com/knative-sandbox/kn-plugin-func/blob/main/templates/go/manifest.yaml. It is additive to what is in the `DefaultBuilderImages` map. Those are builder images, and these are buildpacks that are applied to the builder image at build time.